content/_data/changelogs/weekly.yml

```
@@ -25013,7 +25054,33 @@
           when deserializing XML files"
         message: |-
           Ignore values with incorrect types when deserializing collections and maps in XML files.
-
+      - type: bug
+        category: bug
+        pull: 9739
+        issue: 72979
+        authors:
+          - debayangg
+        pr_title: "[JENKINS-72979] Remove trailing space from Windows agent secret file instructions"
+        message: |-
+          Remove trailing space from Windows agent secret file instructions.
+      - type: bug
+        category: bug
+        pull: 9810
+        issue: 73835
+        authors:
+          - dwnusbaum
+        pr_title: "[JENKINS-73835] Do not allow builds to be deleted while they are still running and ensure build discarders run after builds are fully complete"
+        message: |-
+          Do not allow builds to be deleted while they are still building.
+          Ensure build discarders only process builds which have fully completed.
+      - type: rfe
+        category: developer
+        pull: 9813
+        authors:
+          - Vlatombe
+        pr_title: "Create a new taglib to capture the save/apply bottom bar"
+        message: |-
+          Developer: a new taglib <code><f:saveApplyBar/></code> is now available for configuration forms.
   # pull: 9844 (PR title: Update dependency org.jenkins-ci.plugins:matrix-project to v839)
```

Add the missing items that would have been included as comments in case we need them for future reference

Remove the extra XML wrapper around saveApplyBar so that we don't need to escape the characters.
```suggestion
          Developer: Add a new taglib <code>saveApplyBar</code> for configuration forms.
  # pull: 9870 (PR title: Update dependency org.jenkins-ci.plugins:script-security to v1365)
  # pull: 9868 (PR title: Update dependency io.jenkins.plugins:json-api to v20240303-101.v7a_8666713110)
  # pull: 9866 (PR title: Update dependency io.jenkins.plugins:asm-api to v9.7.1-97.v4cc844130d97)
  # pull: 9865 (PR title: Update dependency sass to v1.79.5)
  # pull: 9864 (PR title: Update dependency postcss-preset-env to v10.0.7)
  # pull: 9863 (PR title: Update babel monorepo to v7.25.8)
  # pull: 9860 (PR title: Update dependency org.jenkins-ci.plugins:mailer to v489)
  # pull: 9859 (PR title: Update dependency org.jenkins-ci.plugins:display-url-api to v2.209.v582ed814ff2f)
  # pull: 9844 (PR title: Update dependency org.jenkins-ci.plugins:matrix-project to v839)
  # pull: 9809 (PR title: Add tests for empty directory in tar and zip archives)
```